### ISSUE TITLE
Lower Resnet50 accuracy with ONNXRuntime

### ISSUE THREAD

**Issue by yeshuj** [microsoft/onnxruntime#10021]
**Describe the bug**
Ater exporting keras resnet50 model to onnx and running it with onnxruntime, i get a lower accuracy than what keras reported : https://keras.io/api/applications/

With onnxruntime I get 74.7% while I expect to get around 74.9%. I'm using the same imagenet data. Is it expected to observe this accuracy drop?

I did not quantize the model.

**Comment by yufenglee** [comment]
@yeshuj, just want to double confirm that you repro keras's reported accuracy with keras. Could you get those which get wrong prediction with ORT and compare their score with keras?

**Comment by snnn** [comment]
Have tried https://github.com/mlcommons/inference ? The last month I checked our accuracy on resnet50 was pretty good.

**Comment by yeshuj** [comment]
@snnn The prepeocessing steps I followed are same, except that I used OPSET 11 instead of 8 when exporting tensorflow model to ONNX for resnet50, does that impact the accuracy?

**Comment by snnn** [comment]
It could. But I tried some newer opsets.  I didn't found problems.  You may try to reproduce the problem with the mlperf scripts first.  The model conversion script  is at: https://github.com/mlcommons/inference/blob/master/vision/classification_and_detection/tools/convert-to-onnx.sh Please feel free to change the opset number. Once you successfully reproduce it, please let us know which opset number you used. Then we will start to investigate. 
BTW, in order to get an accurate number, you need to use the full validation dataset(50,000 images).  Otherwise the accuracy number would be different.

**Comment by snnn** [comment]
This issue has been automatically closed as 'not planned' because it has been marked as 'stale' for more than 30 days without activity. If you believe this is still an issue, please feel free to reopen it.
